packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -339,45 +345,58 @@ export const DatePicker = ({
         aria-live="polite"
       >
         <FocusTrap focusTrapOptions={focusTrapOptions}>
-          <StyledDayPicker
-            firstDayOfWeek={1}
-            todayButton="Jump to today"
-            onTodayButtonClick={(day) => {
-              if (isDateDisabled(day, disabledDays)) {
-                return
-              }
+          <div>
+            {jumpToToday && (
+              <StyledJumpToToday
+                variant={BUTTON_VARIANT.TERTIARY}
+                size={COMPONENT_SIZE.SMALL}
+                aria-label="Jump to today"
+                aria-disabled="false"
```

Comment:
Are these two props needed? (The label matches the button text, and `aria-disabled` value is hard-coded to false.)

Reply:
Hmm no I don't think so they were in NSS for some reason. I'll remove.